src/rubrix/client/sdk/token_classification/models.py

```diff
@@ -118,6 +118,7 @@ def to_client(self) -> ClientTokenClassificationRecord:
             status=self.status,
             metadata=self.metadata or {},
             metrics=self.metrics or {},
```

Comment:
Not related to this PR, but I think this should be None
```suggestion
            metrics=self.metrics or None,
```